tests/minimal_init.lua

```diff
@@ -20,9 +20,39 @@ for lang, _ in vim.fs.dir("queries") do
     table.insert(langs, lang)
   end
 end
-require("nvim-treesitter.configs").setup({
-  ensure_installed = langs,
-  sync_install = true,
-})
--- this needs to be run a second time to make tests behave
-require("nvim-treesitter").setup()
+local master_nvim_ts, configs = pcall(require, "nvim-treesitter.configs")
+if master_nvim_ts then
+  configs.setup({
+    ensure_installed = langs,
+    sync_install = true,
+  })
+  -- this needs to be run a second time to make tests behave
+  require("nvim-treesitter").setup()
+
+  vim.api.nvim_create_user_command("RunTests", function(opts)
+    local path = opts.fargs[1] or "tests"
+    require("plenary.test_harness").test_directory(
+      path,
+      { minimal_init = "./tests/minimal_init.lua" }
+    )
+  end, { nargs = "?" })
+else
+  vim.api.nvim_create_user_command("RunTests", function(opts)
+    local path = opts.fargs[1] or "tests"
+    require("nvim-treesitter.install").install("all", { skip = { installed = true } }, function()
```

Comment:
Does the new API support installing just the list of languages that we support?

Reply:
Just took another look and it does. We can pass `langs` here.

As an aside, there's an issue with installing `norg` parser with `main` branch - it's not present there. If `master` is run first, the compiled parsers are visible to neovim, but if `main` is run on its own, an annoying issue arises.

Specifically, norg parser relies on c++14 standard. At the very least, default MacOS clang version does not enable required features by default, and fails to compile the parser. Providing `-std=c++13` results in `error: invalid argument '-std=c++14' not allowed with 'C'`.

Latest `clang` from homebrew works ok, and `gcc-13` also doesn't bat an aye.

In order to use a different compiler, `CC` environment variable can be used, e.g.:
```sh
CC=gcc-13 ./run_tests.sh --test-main
```

I just pushed a separate commit that adds neorg parser configuration and uses `langs` instead of `"all"` to install only the languages aerial handles.